.circleci/config.yml

```diff
@@ -796,7 +750,19 @@ workflows:
             equal: ["UNKNOWN", << pipeline.parameters.study_key >>]
     jobs:
       - deploy-stored-build-job:
+          name: deploy-stored-build-<< pipeline.parameters.study_key >>-<< pipeline.parameters.deploy_env >>
           study_key: << pipeline.parameters.study_key >>
+      # After deploy, run Playwright E2E tests against Dev or Test but not for Staging and Production
+      - playwright-build:
+          env: << pipeline.parameters.deploy_env >>
+          requires:
+            - deploy-stored-build-job
+      - playwright-test:
```

Comment:
If we want to restrict Playwright jobs to only rc and hotfix branches, then just add this branch filter: `<<: *filter-rc-hotfix-branch`.

Reply:
Let's run playwright regardless of branch for dev/test deploys.  If it's noisy, we can revisit.  I like getting instant feedback on branches!